tests/suites/charmhub/download.sh

```diff
@@ -10,7 +10,7 @@ run_charmhub_download() {
 	check_contains "${output}" 'Fetching charm "juju-qa-test"'
 
 	juju deploy "${TEST_DIR}/juju-qa-test.charm" juju-qa-test
-	juju wait-for application --timeout=15m juju-qa-test
+	wait_for "juju-qa-test" "$(active_idle_condition "juju-qa-test")" 900
```

Comment:
I kept the original timeout. Remember that's an absolute maximum, if the app is ready earlier, then this will exit early. It's possible that it could take this long if you're on a slow machine and have bad luck.

These tests are not being run on Jenkins AFAIK.